vector/src/main/java/im/vector/riotx/features/home/room/list/RoomListFragment.kt

```diff
@@ -78,6 +79,19 @@ class RoomListFragment : VectorBaseFragment(), RoomSummaryController.Listener, O
         injector.inject(this)
     }
 
+    override fun getMenuRes() = R.menu.room_list
```

Comment:
I think it should be handled in the host activity, as we use RoomList in different contexts 
(for handling incoming share as exemple)

Reply:
I've changed the way it's done. Please review changes